module/renderer/src/androidUnitTest/kotlin/org/cru/godtools/shared/renderer/content/RenderTabsPaparazziTest.kt

```diff
@@ -0,0 +1,63 @@
+package org.cru.godtools.shared.renderer.content
+
+import kotlin.test.Test
+import org.cru.godtools.shared.renderer.BasePaparazziTest
+import org.cru.godtools.shared.tool.parser.model.Dimension
+import org.cru.godtools.shared.tool.parser.model.Gravity
+import org.cru.godtools.shared.tool.parser.model.Image
+import org.cru.godtools.shared.tool.parser.model.Spacer
+import org.cru.godtools.shared.tool.parser.model.Tabs
+import org.cru.godtools.shared.tool.parser.model.Text
+
+class RenderTabsPaparazziTest : BasePaparazziTest() {
+
+    @Test
+    fun `RenderTabs() - Tabs`() = contentSnapshot {
+        RenderContentStack(
+            listOf(
+                Text(text = "Render Tabs"),
+                Spacer(),
+                Tabs(
+                    tabs = listOf(
+                        Tabs.Tab(
+                            label = Text(text = "1"),
+                            content = listOf(
+                                Spacer(),
+                                Text(
+                                    text = "Tab 1 Content",
+                                    textAlign = Text.Align.CENTER
+                                ),
+                                Image(
+                                    manifest,
```

Comment:
How you should actually do this is by properly linking children to their parents.

```
Tabs(manifest) {
    listOf(
        Tab(it) {
            listOf(
                Image(it, resource = "bruce")
            )
        }
    )
}
```

Reply:
the model is setup so that any intermediate model that supports it can override styles for all of it's descendants. But it only really works if you have the proper linkage between children and their parents